sdk/batch/batch-rest/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.0-beta.3 (2025-07-15)
+
+### Features Added
+
+- Updated model names to improve consistency and clarity across the SDK.
+
```

Comment:
I think it would be good to list out all the name changes, see https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/batch/Azure.Compute.Batch/CHANGELOG.md